core/silkworm/state/intra_block_state.cpp

```diff
@@ -312,10 +315,11 @@ void IntraBlockState::revert_to_snapshot(const IntraBlockState::Snapshot& snapsh
 
 void IntraBlockState::finalize_transaction() {
     for (auto& x : storage_) {
```

Comment:
More CXX17 style and makes clear we're iterating a map.
Also helpful in write_to_db etc.

```suggestion
    for (auto& [address, storage] : storage_) {
        for (const auto& [a, b] : storage.current) {
            storage.committed[a].original = b;
        }
        storage.current.clear();
    }
```

Reply:
Cool, I didn't know about this feature